core/tools/nginx/nginx.conf

```diff
@@ -164,8 +164,10 @@ http {
         location / {
             root /home/pi/frontend;
             try_files $uri $uri/ /index.html;
+        }
+
+        location ~* /.(html|css|js)$ {
             expires -1;
-            add_header Cache-Control no-store;
```

Comment:
I suspect we need this. it seems to be caching files it shouldn't:
![image](https://user-images.githubusercontent.com/4013804/160208161-f4a28826-42dc-4d38-bdd8-418b16a73ca7.png)

Reply:
Good catch. Thought the "expires" was enough. Rebuilding.